### ISSUE TITLE
The gamesir 4.2.9 still does not work

### ISSUE THREAD

**Issue by Pixel4aUser** [gamesir123/app#100]
Dear gamesir,

App sill does not work for andorid 12 nor 13.
If any of you got it work on any andorid emulator, please tell me.

So far, Mumu player did not work. 

Environemnt: Google Pixel 4a.

**Comment by Drdynasty** [comment]
It works for me, my only issues is I don't use there controllers I use the vx2 aimbox and for some reason I still havnt found a solution to get the app for that to work! Devoplers are getting lazy.......

**Comment by Pixel4aUser** [comment]
> It works for me, my only issues is I don't use there controllers I use the vx2 aimbox and for some reason I still havnt found a solution to get the app for that to work! Devoplers are getting lazy.......

yeah they are becoming lazy i guess. At least I want to know if they are wroking on it or not. 

I use VX aim box 2 as well. I think it is because pixel auto updating system to android 13 that makes it not work. 
Gamesir is part of Xiomi Ecochain so they should know better tho...

**Comment by Drdynasty** [comment]
Yeah because that update they released was a total different update from the vx it was for the controllers which i think is ridiculous update all not just the one l, well I'll be opting in for a refund since there device is unusable
